front-end/src/components/search/QueryBox.js

```diff
@@ -44,8 +35,7 @@ const QueryBox = (props) => {
 
     fetch(requestURL + new URLSearchParams({
         q:props.searchQuery,
-        df:`text_${props.currentLang}_txt`,
-        'q.op': "AND"
```

Comment:
This looks like it reverts CU-21prurv which got committed in PR #24

Reply:
Resolved this one myself. This was the only blocker for merging, so I'm proceeding.